packages/fast-components-react-msft/src/button/button.props.ts

```diff
@@ -0,0 +1,29 @@
+import * as React from "react";
+import { IButtonHandledProps } from "@microsoft/fast-components-react-base";
+import { IManagedClasses, IMSFTButtonClassNameContract } from "@microsoft/fast-components-class-name-contracts-msft";
+
+export interface IMSFTButtonHandledProps extends IButtonHandledProps {
+    /**
+     * The lightweight justified option
+     */
+    justified?: boolean;
+
+    /**
+     * The lightweight option
+     */
+    lightweight?: boolean;
+
+    /**
+     * The outline option
+     */
+    outline?: boolean;
```

Comment:
Have we considered secondary?

Reply:
Secondary used to be used for the non-primary, but that is now default with primary being specified. Calling this secondary would be confusing in my opinion. Outline is descriptive of what this visually looks like.

Reply:
Primary used to be enumerated for button with secondary being the default. I think we want that swapped. Primary for the most part is almost always what people have expected as the default and I think in previous conversations with Jason he had mentioned that primary becomes the default...this may take some thinking around the classNames, but I think that for the most part when adding a button people expect primary. This will be helpful if we are using it to create a CTA as well as that default is primary as well.

Reply:
This directly contradicts a conversation I had with Jason, so we need clarification

Reply:
Yep - We may have reversed course due to other MS DL's base button styles, but we should definitely seek to clarify.